lib/travis/addons/handlers/github_status.rb

```diff
@@ -10,12 +10,13 @@ class GithubStatus < Base
         EVENTS = /build:(created|started|finished|canceled|restarted)/
 
         def handle?
+          return true if installation?
           Addons.logger.error "No GitHub OAuth tokens found for #{object.repository.slug}" unless tokens.any?
           tokens.any?
         end
 
         def handle
-          run_task(:github_status, payload, tokens: tokens)
+          run_task(:github_status, payload, tokens: tokens, installation: installation_github_id)
```

Comment:
`installation_github_id` does not sound right to me. This is the ID of the GitHub Apps installation, right? How about something like: `github_apps_installation_id` (wordy, but more descriptive)?